src/ng/http.js

```diff
@@ -153,6 +211,11 @@ function $HttpProvider() {
    *     - **`defaults.headers.put`**
    *     - **`defaults.headers.patch`**
    *
+   * - **`defaults.paramSerializer`** - {string|function([paramsObject])} - A function used to prepare string representation
```

Comment:
I think the type should be `{string|function(Object<string,string>):string}`, no?